examples/3_automatic_method_suggestion.ipynb

```diff
@@ -89,7 +89,7 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "100%|██████████| 13/13 [01:56<00:00,  8.96s/it]\n"
+      "100%|██████████| 14/14 [01:57<00:00,  8.39s/it]\n"
```

Comment:
Possibly got faster due to #154. I'm seeing `polydiff` takes 30-35 seconds to optimize now. It was formerly nearly a minute.